omero/developers/testing.rst

```diff
@@ -113,17 +99,10 @@ explicitly via:
 
 Results are placed in ``components/<component>/target/reports``.
 
+
 Individual tests
 """"""""""""""""
 
-Alternatively, you can run individual tests which you may currently be
-working on. This can be done using the ``test`` target. For example:
-
-::
-
-    ./build.py -f components/tools/OmeroJava/build.xml test -DTEST=integration/chown/PermissionsTest
-    ./build.py -f components/tools/OmeroPy/build.xml test -DTEST=test/integration/test_admin.py
```

Comment:
Did this information (integration tests for Python) get moved anywhere? I might be missing something.

Reply:
This is done via pytest. 
I had to make a no-op for ``test`` target since it was broken for unit-test.
if user wants to run specific tests they go to the corresponding component and use either ``gradle`` or ``pytest``

Reply:
Sure, I just mean in terms of the docs update.

Reply:
For Java, the same example is under the Java  ``Individual test``
For Python, the example is there but admittedly under the ``-s`` option
I will add a section ``Individual test`` under python to make it clearer that the info is not lost

Reply:
done.
Actually @will-moore update was not correct since the example was listing under ``OmeroWeb`` not ``OmeroPy``